src/lang/en-US.json

```diff
@@ -851,6 +880,7 @@
   "title_explorer": "Data Explorer",
   "title_meta": "Meta",
   "title_records": "Records",
+  "title_api": "Get your Opendota API Key - stats just a GET call away.",
```

Comment:
OpenDota

Reply:
I might just make this a more generic title string like "OpenDota API Key Registration"